BookPlayer/Library/ItemListViewController.swift

```diff
@@ -380,6 +400,27 @@ extension ItemListViewController {
             self.tableView.contentInset.bottom = rootViewController.miniPlayerIsHidden ? 0.0 : 88.0
         }
     }
+
+    @objc func selectButtonPressed(_ sender: Any) {
+        if self.tableView.numberOfRows(inSection: 0) == (self.tableView.indexPathsForSelectedRows?.count ?? 0) {
+            for row in 0..<self.tableView.numberOfRows(inSection: 0) {
+                self.tableView.deselectRow(at: IndexPath(item: row, section: 0), animated: true)
+            }
+
+            self.bulkControls.moveButton.isEnabled = false
```

Comment:
let's move all ocurrences in this file of `self.bulkControls.moveButton.isEnabled =<flag>` and `self.bulkControls.trashButton.isEnabled = <flag>` and let's add them to the function `updateSelectButtonTitle` ( we should probably rename it to `updateSelectionStatus` or something like that)